scrapy/utils/test.py

```diff
@@ -71,11 +72,11 @@ def get_crawler(spidercls=None, settings_dict=None, prevent_warnings=True):
     return runner.create_crawler(spidercls or Spider)
 
 
-def get_pythonpath():
+def get_pythonpath() -> str:
     """Return a PYTHONPATH suitable to use in processes so that they find this
     installation of Scrapy"""
     scrapy_path = import_module('scrapy').__path__[0]
-    return os.path.dirname(scrapy_path) + os.pathsep + os.environ.get('PYTHONPATH', '')
+    return str(Path(scrapy_path).parent) + os.pathsep + os.environ.get('PYTHONPATH', '')
```

Comment:
Not related to this PR, but this code looks suspicious; I wonder why path is not just joined, and os.pathsep is used instead.

Reply:
`os.pathsep` is a separator of paths (: or ;) as opposed to `os.sep`. It's a list of paths, not a single path.